tool/tsh/common/tsh.go

```diff
@@ -4507,6 +4507,35 @@ func printStatus(debug bool, p *profileInfo, env map[string]string, isActive boo
 	fmt.Printf("\n")
 }
 
+func isOktaRole(role string) bool {
+	return strings.Contains(role, teleport.OktaReviewerRoleContext) || strings.Contains(role, teleport.OktaAccessRoleContext)
+}
+
+func rolesToString(debug bool, roles []string) string {
+	sort.Strings(roles)
+	var nonOktaRoles, oktaRoles []string
+	for _, role := range roles {
+		if isOktaRole(role) {
+			// Keep okta roles and the end of list.
```

Comment:
Nit: I get what you mean by this comment, but I think it's slightly confusing. IMO the wording and placement of the comment suggests you're appending Okta roles to `oktaRoles` and prepending non Okta roles to `oktaRoles`, which is obviously not the case as you append non Okta roles to `nonOktaRoles` right below. I'd consider moving this comment to be above line 4535 for example, as then it makes it clear that you want to display Okta roles last.

```suggestion
			// Keep okta roles at the end of list.
```